spec/api/search_spec.rb

```diff
@@ -170,28 +165,24 @@ def check_sort(sort_key, sort_order, expected_thread_indexes)
 
         it "by date" do
           asc_order = [0, 1, 2, 3, 4, 5]
-          check_sort("date", "asc", asc_order)
-          check_sort("date", "desc", asc_order.reverse)
+          check_sort("date", asc_order.reverse)
```

Comment:
I'd recommend inlining the expected ordering (like other test points below), instead of defining it and then reversing it.